vscode-ext/src/StatusBarComponent.ts

```diff
@@ -1,15 +1,29 @@
-import { StatusBarItem } from "vscode";
+import { StatusBarItem, Range } from "vscode";
 import * as vscode from 'vscode';
 import { StatusBarViewConfig } from "./Config";
-import { ServerStatus } from "./CaesarClient";
-import { Verifier } from "./Verifier";
+import { ServerStatus, VerifyResult } from "./CaesarClient";
+import { DocumentMap, Verifier } from "./Verifier";
 import { ConfigurationConstants } from "./constants";
 
+
+const runningTooltipBase =
+    new vscode.MarkdownString(
+        "[Stop Caesar](command:caesar.stopServer)\n\n" +
+        "[Restart Caesar](command:caesar.restartServer)"
+        , true);
+
+const stoppedTooltipBase =
+    new vscode.MarkdownString(
+        "[Start Caesar](command:caesar.startServer)", true);
+
 export class StatusBarComponent {
 
     private enabled: boolean;
-    private status: ServerStatus = ServerStatus.Stopped;
+    private verifyStatus: DocumentMap<[Range, VerifyResult][]> = new DocumentMap();
+    private serverStatus: ServerStatus = ServerStatus.NotStarted;
     private view: StatusBarItem;
+    private tooltipBase: vscode.MarkdownString = stoppedTooltipBase;
+    private tooltipExtension: vscode.MarkdownString = new vscode.MarkdownString();
```

Comment:
Maybe rename this to `tooltipStatusText`? The current name is a bit confusing to me.